CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+# Changelog DeviceLocation
+
+## Table of Contents
+
+- [v0.2.0](#v020)
+- [v0.1.0](#v010)
+
+**Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
+
+# v0.2.0
+
+**This is the second alpha version of the DeviceLocation API family.**
+
+- API [definition](https://github.com/camaraproject/DeviceLocation/tree/release-0.2.0/code/API_definitions) **with inline documentation**.
+
+## Please note:
+
+- **This release contains significant breaking changes compared to v0.1.0, and it is not backward compatible**
+  - Especially a lot of the parameter names changed in line with the agreed glossary within CAMARA Commonalities
+- This is an alpha version, it should be considered as a draft.
+- There are bug fixes to be expected and incompatible changes in upcoming versions. 
+- The release is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
+
+
+## What's changed
+
+* Second alpha version of `location-verification` API, v0.2.0, renamed from `location`, with main changes:
+
+    - Input property `ueId` evolved to `device`, following CAMARA guidelines.
+    - New input parameter `area`, considering initially circular areas, and deprecating previous `latitude`, `longitude` and `accuracy` properties.
+    - New input parameter `maxAge`.
+    - Response adds new values `UNKNOWN` and `PARTIAL` for `verificationResult`. It also adds properties `matchRate` and `lastLocationTime`.
+    - Security schemas and scopes aligned with commonalities guidelines.

```

Comment:
@jlurien - Do we have an agreement on this topic in commonalities? I see [this](https://github.com/camaraproject/Commonalities/issues/53) issue still open?

Reply:
We still have to refine some details in Commonalities, but the most important applied change is the use of openIdConnect for securitySchemes, which was approved by the [TSC](https://github.com/camaraproject/Governance/blob/main/documentation/MeetingMinutes/TSC/TSC-2023-10-19-Meeting-Minutes.md#commonalities--icm). Hopefully we can advance on closing the other details before the Release Candidate is finally approved. If further changes come, we'll have to apply them in future releases.